spark-jobs/src/main/resources/META-INF/services/za.co.absa.hyperdrive.ingestor.api.transformer.StreamTransformerFactoryProvider

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright 2018 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+za.co.absa.enceladus.conformance.HyperConformanceProvider
```

Comment:
What is the point of this file or its content?

Reply:
Hyperdrive uses (or will use) the Java Service Provider API to load plugins like this one. The Java utility `ServiceLoader` checks the files under `META-INF/services`, whose filenames have to be interfaces (traits) and each line of the file must be a fully qualified name of an implementation of the interface.
If you remove this file, `HyperConformanceSuite` is going to fail.

Reply:
Spark uses the Java Service Provider API as well, e.g. you can search for the files `org.apache.spark.sql.sources.DataSourceRegister`

Reply:
Wow, thanks for the explanation.